http/_negotiation/language.ts

```diff
@@ -140,5 +144,5 @@ export function preferredLanguages(
   return priorities
     .filter(isQuality)
     .sort(compareSpecs)
-    .map((priority) => provided[priorities.indexOf(priority)]);
+    .map((priority) => provided[priorities.indexOf(priority)]!);
```

Comment:
Ditto. Please justify this non-null assertion.

Reply:
Same here